source/managing_domain/03_analyzing_domain_usage/index.rst

```diff
@@ -11,6 +11,11 @@ There are three main views to analyze your domain, trying to answer three differ
 
 In the Users and Actions view, you can select a period (default is the last week).
 
+.. admonition:: Note
+   :class: note
+
+   If the domain has subdomains, a feature allowing the monitoring of these subdomains through the main domain's Analytics can be activated on demand. When this feature is activated, a "Select domain" filter is displayed at the top of the dashboard. In this filter, tick "Subdomains" to take them into account in the monitoring dashboards. Note that the monitoring of subdomains can only be available for the Users, Actions and Backoffice monitoring dashboards.
```

Comment:
Technically the "Data" monitoring dashboard will include the subdomain data, I'm not sure this is on purpose

Reply:
nope, not on purpose, I will check that in the code. I also need to remove the filter on the Quality dashboard.